example/lib/main.dart

```diff
@@ -193,7 +194,8 @@ class _MyAppState extends State<MyApp> {
                   _btnSection(),
                   _engineSection(),
                   _futureBuilder(),
-                  _buildSliders()
+                  _buildSliders(),
+                  _getMaxSpeechInputLengthSection(),
```

Comment:
since this is only supported on android, you should check for that.  `if (isAndroid) `

Reply:
Done, please let me know if I missed something!